### PR TITLE
Update links to HTTPS where possible

### PR DESCRIPTION
**Pull request by Gitoffthelawn**

### CODE REVIEW

**Review by kmilos — DISMISSED** [review]
Thanks!

**Comment by kmilos** [comment]
Please rebase.

**Comment by Gitoffthelawn** [comment]
How do I accomplish that task?  Can you advise?

Because the changes are minor, I can obviously just edit the current main branch and implement the same changes, but I would like to learn how to do what you describe (for situations where the changes are not so minor).

**Comment by kmilos** [comment]
> How do I accomplish that task? Can you advise?

You can try the suggested links under the "This branch has conflicts..." below.

**Comment by Gitoffthelawn** [comment]
Thanks.  It wound up requiring manual editing.  It surprises me that GitHub doesn't handle this situation more efficiently.  But no big deal for small changes.

File: README.md [hunk 1]
```diff
@@ -271,9 +271,9 @@ The following Exiv2 features require external libraries:
 
 | Feature                  | Package  | Default | To change default           | Availability |
 |:------------------------ |:-------- |:-------:|:--------------------------- |:------------ |
-| PNG image support        | zlib     | ON      | -DEXIV2\_ENABLE\_PNG=OFF    | [http://zlib.net/](http://zlib.net/) |
-| XMP support              | expat    | ON      | -DEXIV2\_ENABLE\_XMP=OFF    | [https://libexpat.github.io/](https://libexpat.github.io/)<br/>Use _**Expat 2.2.6**_ and later |
-| Natural language system  | gettext  | OFF     | -DEXIV2\_ENABLE\_NLS=ON     | [http://www.gnu.org/software/gettext/](http://www.gnu.org/software/gettext/) |
+| PNG image support        | zlib     | ON      | -DEXIV2\_ENABLE\_PNG=OFF    | [https://zlib.net/](https://zlib.net/) |
+| XMP support              | expat    | ON      | -DEXIV2\_ENABLE\_XMP=OFF    | [https://libexpat.github.io/](https://libexpat.github.io/)/<br/>Use _**Expat 2.2.6**_ and later |
```
Comment (kmilos) [inline README.md:275]:
```suggestion
| XMP support              | expat    | ON      | -DEXIV2\_ENABLE\_XMP=OFF    | [https://libexpat.github.io/](https://libexpat.github.io/)<br/>Use _**Expat 2.2.6**_ and later |
```

**Review by kmilos — APPROVED** [review]

**Comment by kmilos** [comment]
> It wound up requiring manual editing.

Yep, usually does when there is a conflict.

**Comment by Gitoffthelawn** [comment]
Thank you.  What surprised me is that one of the changes was on its own line, but GitHub complained.  Any idea why?

**Pull request merged**